airbyte-integrations/connectors/destination-postgres/src/main/java/io/airbyte/integrations/destination/postgres/typing_deduping/PostgresSqlGenerator.java

```diff
@@ -149,21 +148,6 @@ public Sql createTable(final StreamConfig stream, final String suffix, final boo
     return Sql.concat(statements);
   }
 
-  @Override
-  protected List<String> createIndexSql(final StreamConfig stream, final String suffix) {
-    if (stream.getDestinationSyncMode() == DestinationSyncMode.APPEND_DEDUP && !stream.getPrimaryKey().isEmpty()) {
-      return List.of(
-          getDslContext().createIndex().on(
```

Comment:
in `createTable`, we're already creating an index on `pk, cursor, extracted_at`. Creating an index on just `pk` is redundant.